torchrl/envs/transforms/transforms.py

```diff
@@ -1522,8 +1531,13 @@ def __init__(
         self.register_buffer("scale", scale.clamp_min(1e-6))
 
     def _apply_transform(self, reward: torch.Tensor) -> torch.Tensor:
-        reward.mul_(self.scale).add_(self.loc)
-        return reward
+        if self.standard_normal:
+            loc = self.loc
+            scale = self.scale
+            return (reward - loc) / scale
+        else:
+            reward.mul_(self.scale).add_(self.loc)
```

Comment:
I know we were doing it like this before, but I think that in general we should avoid in-place changes (say someone has a reward computed through a model and wants to differentiate through it, things might break).
What about 
```python
reward = rewad * scale + loc
```